xUnitTests/LoopringGraphTests/TestNFT.cs

```diff
@@ -67,5 +69,16 @@ public async void GetCollectionNFTs(string tokenAddress)
             var nfts = await service.GetCollectionNFTs(tokenAddress);
             Assert.NotEmpty(nfts);
         }
+
+        [Fact]
+        public async void GetNftHolders_WhenPassingValidNftId_ExpectListOfHolders()
+        {
+            string nftId = "0xf5ca5e167819d449270dae6fa22208085a2c03d9-0-0xcd56d4e1dba6edf425aa3e0f901d9f1645887539-0x33f5b90cdf6271026f4a0e4817de618268e6e391320a4ad834ea54fd87032fcb-2";
+            List<NftHolder> holders = await service.GetNftHolders(nftId);
+            Assert.NotEmpty(holders);
+
+            Assert.IsType<NftHolder>(holders[0]);
+            Assert.Equal(25, holders.Count());
```

Comment:
Checking for this hardcoded will very likely break in the future, don't you think? I mean you're actually testing two things here:

* that the default page size aka `first` parameter of `GetNftHolders` is 25
* that this specific Nft has more than 25 holders on L2

Again, I don't see how this improves the test.

Reply:
From where the query is right now, it will never fail because that specific nft will always get more than 25 holders. So the result of this query will always give us 25. If we change the pageSize for instance etc we would have a failing test and therefore update the test.

Reply:
Fixed tests